src/hotspot/share/gc/z/zDriver.cpp

```diff
@@ -98,6 +102,10 @@ class VM_ZOperation : public VM_Operation {
     ZStatSample(ZSamplerJavaThreads, Threads::number_of_threads());
   }
 
+  virtual bool allow_coalesced_vm_operations() const {
+    return false;
+  }
+
```

Comment:
Per explicitly wanted skip_thread_oop_barriers grouped with needs_inactive_gc_locker. But I should remove allow_coalesced_vm_operations; it is no longer used since I have removed VM operation coalescing completely already.

Reply:
I agree with Per, but I also wanted to move allow_coalesced_vm_operations. But now your removing it, so it becomes a moot point.